app/resources/locale/en/en.po

```diff
@@ -73,6 +92,10 @@ msgstr "Basque"
 msgid "Binary"
 msgstr "Binary"
 
+#: app/scripts/services/tools.js:159
+msgid "Bitstream not found, build first your project"
+msgstr "Bitstream not found, build first your project"
+
```

Comment:
"Bitstream not found. First build your project" with or without a dot at the end

Reply:
👌